client/components/mma/switch/SwitchOptions.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import { ReactRouterDecorator } from '../../../../.storybook/ReactRouterDecorator';
+import type { PaidSubscriptionPlan } from '../../../../shared/productResponse';
+import { contribution } from '../../../fixtures/productDetail';
+import SwitchContainer from './SwitchContainer';
+import SwitchOptions from './SwitchOptions';
+
+export default {
+	title: 'Pages/SwitchOptions',
+	component: SwitchContainer,
+	decorators: [ReactRouterDecorator],
+	parameters: {
+		layout: 'fullscreen',
+		reactRouter: {
+			state: { productDetail: contribution },
+			container: <SwitchContainer />,
+		},
+	},
+} as ComponentMeta<typeof SwitchContainer>;
+
+export const AboveThreshold: ComponentStory<typeof SwitchOptions> = () => (
+	<SwitchOptions />
+);
+
+export const BelowThreshold: ComponentStory<typeof SwitchOptions> = () => (
+	<SwitchOptions />
+);
+
+// @ts-ignore: this method is not in our current version of TypeScript
```

Comment:
You'll need to change this to `@ts-expect-error` as I just merged the linting changes 😅

Reply:
Do you think this code is why the story didn't show up in Chromatic? Or is that a coincidence

Reply:
Oh, perhaps it doesn't support `structuredClone` or we need to polyfill it? You could clone the object in a slightly hackier (but well supported) way:

```js
const contributionBelowThreshold = JSON.parse(JSON.stringify(contribution));
```

(You'll see this being done in a couple of the Cypress specs.)